modules/xphoto/src/inpainting_fsr.impl.hpp

```diff
@@ -767,7 +767,7 @@ void inpaint_fsr(const Mat &src, const Mat &mask, Mat &dst, const int algorithmT
                 CV_Error(Error::StsUnsupportedFormat, "Unsupported source image format!");
                 break;
             }
-            src.convertTo(src, CV_8U, 1/257.0);
+            src.convertTo(src, CV_8U, 1/255.0);
```

Comment:
In case if you want to get only  higher 8 bits of value, the multiplier should be 1/256.